qa/suites/rados/singleton/all/mon_memory_target_compliance.yaml

```diff
@@ -0,0 +1,159 @@
+# Fill the top level branch items as appropriate
+branch: wip-sseshasa2-testing-2019-07-30-1825
```

Comment:
everything before roles here doesn't belong in the qa suite - it's just for manually run jobs. When a suite is scheduled, it will have these parameters set for you based on the options passed to teuthology-suite

Reply:
I have removed everything before roles.